example/README.md

```diff
@@ -0,0 +1,342 @@
+        <- success
+    ```
+
+4. Register service for resource_id from [the step 2](#resource-id):
+
+    ```rust
+    fluence run -f 'registerService("echo12D3KooWRgEgxP4qAyUR5jerwKE1rwTLZoAhJ3YwAhpeSMC5pi77")'
+    ```
+
+    output:
+
+    ```bash
+    $ fluence run -f 'registerService("echo12D3KooWRgEgxP4qAyUR5jerwKE1rwTLZoAhJ3YwAhpeSMC5pi77")'
+    Running:
+      function: registerService("echo12D3KooWCWsJLrG6evQKPEsDmKNbaW8NGj5JhVZfHumSvyx1Zw4c")
+      relay: /dns4/kras-05.fluence.dev/tcp/19001/wss/p2p/12D3KooWCMr9mU894i8JXAFqpgoFtx6qnV1LFPSfVc3Y34N4h4LS
+    ... done
+
+    Result:
+
+    true
+    ```
+
+5. Next, we need to register JS service. For that we have Aqua imports and exports in [src/aqua/export.aqua](src/aqua/export.aqua):
+
+    ```
+    import registerProvider from "@fluencelabs/registry/resources-api.aqua"**
+    export registerProvider
+    ```
+
+6. In [src/echo.ts](src/echo.ts) we should pass resource id as cmd argument:
+
+    ```tsx
+    let [success, error] = await registerProvider(process.argv[2], "echo", serviceId);
+    console.log("registration result: ", success);
+    ```
+
+7. So, stop the previous JS client from [the step 5](#running-service) and run
+`npm run start echo12D3KooWRgEgxP4qAyUR5jerwKE1rwTLZoAhJ3YwAhpeSMC5pi77` (Note: resource_id is from [the step 2](#resource-id))
+output:
+
+    ```rust
+    ...
+    Copy this code to call this service:
+    fluence run -f 'echoJS("12D3KooWCmnhnGvKTqEXpVLzdrYu3TkQ3HcLyArGJpLPooJQ69dN", "12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi", "echo", "msg")'
+    registration result:  true
+    ```
+
+
+## Call services with Registry (using resource_id)
+
+1. In [src/aqua/main.aqua](src/aqua/main.aqua) defined `echoAll` to resolve services and call sequentially:
+
+    ```rust
+    func echoAll(resource_id: string, msg: string) -> *string:
+    		-- 2 is the min number of providers we want to find
+        providers <- resolveProviders(resource_id, 2)
+        results: *string
+        for p <- providers:
+            on p.peer_id via p.relay_id:
+                EchoService p.service_id!
+                results <- EchoService.echo(msg)
+        <- results
+    ```
+
+2. Let’s run all registered echo services with only resource id:
+
+    ```bash
+    fluence run -f 'echoAll("echo12D3KooWRgEgxP4qAyUR5jerwKE1rwTLZoAhJ3YwAhpeSMC5pi77", "hi")'
+    ```
+
+    output:
+    ```
+    $ fluence run -f 'echoAll("echo12D3KooWRgEgxP4qAyUR5jerwKE1rwTLZoAhJ3YwAhpeSMC5pi77", "hi")'
+    Running:
+    function: echoAll("echo12D3KooWRgEgxP4qAyUR5jerwKE1rwTLZoAhJ3YwAhpeSMC5pi77", "hi")
+    relay: /dns4/kras-06.fluence.dev/tcp/19001/wss/p2p/12D3KooWDUszU2NeWyUVjCXhGEt1MoZrhvdmaQQwtZUriuGN1jTr
+    ... done
+
+    Result:
+
+    [
+        [
+            "12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi: hi",
+            "12D3KooWCmnhnGvKTqEXpVLzdrYu3TkQ3HcLyArGJpLPooJQ69dN: hi"
+        ]
+    ]
+    ```
```

Comment:
Summary for the whole example that reiterates what was done and why it's needed. Can emphasise what else can be done and how the example might be developed further to help in daily development.